src/bcccp/carpark/Carpark.java

```diff
@@ -97,7 +97,7 @@ public boolean isSeasonTicketValid(String barcode) {
 		ISeasonTicket ticket = seasonTicketDAO.findTicketById(barcode);
 		
 		// TODO implement full validation logic
-		return ticket != null;
+		return true;
```

Comment:
Is this right?
Haven't you invalidated teh isSeasonTicketValid() method by having it always return true?

Reply:
Fixed, reverted to original.